### DISCUSSION TITLE
How do you handle development access for single projects?

### DISCUSSION THREAD

**Discussion by DennisNissenUX** [Budibase/budibase#6938] [General]
Hey guys,

I have been working a lot with budibase in the last few months and I am actually very impressed with the possibilities it provides.

I am currently working as an external consultant for a larger company that wants to upgrade its internal processes. Instead of Excel lists, Power Apps and various third-party software, we want to implement an environment with budibase where all tools, forms etc. are bound in one place and can be further developed there by tech-savvy power users, while the dev team provides the framework. So far it works very well, but with an increasing number of power users and implemented tools, the clarity of the rights becomes a problem.

In my opinion, the involvement of stakeholders and power users in the development process and the possibility for them to further refine their tool after completion is a huge plus point of low-code tools, which budibase also delivers well. The big drawback, however, is that the development access I want to give the respective power users leads to them getting access to all the company's tools, some of which are somewhat sensitive.

If you could distribute the development access for each project like the respective user roles, that would be a big win.

What is your experience with this issue? Has anyone found a good workaround?

I'm looking forward to learning more about budibase. Huge fan already.

**Comment by mike12345567** [comment]
Hi @DennisNissenUX - in the hopefully near future we plan to resolve this issue, we expected this would become a problem at some point and have been working on a solution that will be a premium feature, user groups (https://github.com/Budibase/budibase/pull/6571).

The PR here is just the beginnings of the work (the core of it) and there are still changes to come, however we have plans for this to extend to development access, as well as users, allowing limiting access to groups of apps etc. This is something that we expected to be an issue and we do plan to resolve it but right now we don't have a specific estimate for when this work will be completed and available. If you and your company have specific needs or would like to discuss this, you should reach out to the us: sales@budibase.com or if you'd like to chat you can schedule one here: https://calendly.com/budibase/

**Reply by DennisNissenUX** [reply]
Hey @mike12345567 thank you very much for the comprehensive answer. That helps me for now. We will contact you when the requirement becomes critical.
